src/Components/MyListsHandler.cs

```diff
@@ -685,113 +862,180 @@ private async Task ShowEditIngredientPopup(Ingredient oldIngredient, UserList us
             Button createListButton = new Button
             {
-                Width = 150, 
-                Height = 40, 
-                Background = Brushes.Transparent, 
-                Foreground = (SolidColorBrush)App.Current.Resources["SecondaryBrush"], 
-                FontSize = 14, 
-                FontWeight = FontWeights.Bold,
-                HorizontalContentAlignment = HorizontalAlignment.Center,
-                VerticalContentAlignment = VerticalAlignment.Center,
                 Content = "Create List",
-                BorderBrush = Brushes.Transparent,
-            };
-
-            // Set the corner radius by wrapping the button in a Border
-            Border border = new Border
-            {
-                Width = 150,
-                Height = 40,
+                Margin = new Thickness(25, 30, 10, 10),
                 HorizontalAlignment = HorizontalAlignment.Left,
-                Background = (SolidColorBrush)App.Current.Resources["TertiaryBrush"],
-                CornerRadius = new CornerRadius(20),
-                Child = createListButton
+                Style = (Style)App.Current.Resources["ExpandButtonStyle"] 
             };
 
-            // Attach the click event handler
+            // Attach the click event handler for the Create List button
             createListButton.Click += CreateListButton_Click;
 
-            // Add the border (with button) to the content area
-            contentArea.Children.Add(border);
+            // Add the Create List button to the content area
+            parentPanel.Children.Add(createListButton);
         }
 
+
         /// <summary>
         /// Popup for moving an ingredient from list to list
         /// </summary>
-        private async Task ShowMoveIngredientPopup(string currListName, Ingredient ingredient)
+        private async Task ShowMoveIngredientPopup(UserList currList, Ingredient ingredient, StackPanel ingPanel)
         {
+            string currListName = currList.GetListName();
+            SolidColorBrush background = (SolidColorBrush)App.Current.Resources["PrimaryBrushB"];
+            SolidColorBrush foreground = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+            SolidColorBrush buttonForeground = (SolidColorBrush)App.Current.Resources["PrimaryBrushB"];
+            SolidColorBrush buttonBackground = (SolidColorBrush)App.Current.Resources["SecondaryBrushB"];
+
+            int boxHeight = 30;
+            int boxWidth = 400;
+            int headingFont = 20;
+
             // Create a new Window for the Move Ingredient popup
             Window moveWindow = new Window
             {
-                Title = "Move Ingredient",
-                Width = 300,
-                Height = 200,
-                WindowStartupLocation = WindowStartupLocation.CenterScreen
+                Title = "Moving " + ingredient.GetName(),
+                Width = 500,
+                Height = 300,
```

Comment:
The popup dialog has way too much space at the bottom.
![image](https://github.com/user-attachments/assets/e40e7121-863a-457c-b423-95e3706ff00c)